pkg/apis/core/v1/conversion.go

```diff
@@ -357,88 +357,6 @@ func Convert_v1_Secret_To_core_Secret(in *v1.Secret, out *core.Secret, s convers
 	return nil
 }
 
-func Convert_core_SecurityContext_To_v1_SecurityContext(in *core.SecurityContext, out *v1.SecurityContext, s conversion.Scope) error {
-	if in.Capabilities != nil {
-		out.Capabilities = new(v1.Capabilities)
-		if err := Convert_core_Capabilities_To_v1_Capabilities(in.Capabilities, out.Capabilities, s); err != nil {
-			return err
-		}
-	} else {
-		out.Capabilities = nil
-	}
-	out.Privileged = in.Privileged
-	if in.SELinuxOptions != nil {
-		out.SELinuxOptions = new(v1.SELinuxOptions)
-		if err := Convert_core_SELinuxOptions_To_v1_SELinuxOptions(in.SELinuxOptions, out.SELinuxOptions, s); err != nil {
-			return err
-		}
-	} else {
-		out.SELinuxOptions = nil
-	}
-	out.RunAsUser = in.RunAsUser
-	out.RunAsGroup = in.RunAsGroup
-	out.RunAsNonRoot = in.RunAsNonRoot
-	out.ReadOnlyRootFilesystem = in.ReadOnlyRootFilesystem
-	out.AllowPrivilegeEscalation = in.AllowPrivilegeEscalation
-	if in.ProcMount != nil {
-		pm := string(*in.ProcMount)
-		pmt := v1.ProcMountType(pm)
-		out.ProcMount = &pmt
-	}
-	return nil
-}
-
-func Convert_core_PodSecurityContext_To_v1_PodSecurityContext(in *core.PodSecurityContext, out *v1.PodSecurityContext, s conversion.Scope) error {
-	out.SupplementalGroups = in.SupplementalGroups
-	if in.SELinuxOptions != nil {
-		out.SELinuxOptions = new(v1.SELinuxOptions)
-		if err := Convert_core_SELinuxOptions_To_v1_SELinuxOptions(in.SELinuxOptions, out.SELinuxOptions, s); err != nil {
-			return err
-		}
-	} else {
-		out.SELinuxOptions = nil
-	}
-	out.RunAsUser = in.RunAsUser
-	out.RunAsGroup = in.RunAsGroup
-	out.RunAsNonRoot = in.RunAsNonRoot
-	out.FSGroup = in.FSGroup
-	if in.Sysctls != nil {
-		out.Sysctls = make([]v1.Sysctl, len(in.Sysctls))
-		for i, sysctl := range in.Sysctls {
-			if err := Convert_core_Sysctl_To_v1_Sysctl(&sysctl, &out.Sysctls[i], s); err != nil {
-				return err
-			}
-		}
-	}
-	return nil
-}
-
-func Convert_v1_PodSecurityContext_To_core_PodSecurityContext(in *v1.PodSecurityContext, out *core.PodSecurityContext, s conversion.Scope) error {
```

Comment:
compare with autogenerated function:
https://github.com/kubernetes/kubernetes/blob/c3e8a434ebef5478f1122bacc1da7423cb23678b/pkg/apis/core/v1/zz_generated.conversion.go#L5513-L5522

(the Convert_... functions being called were the autogenerated ones)